.github/workflows/test-asan.yml

```diff
@@ -3,23 +3,29 @@ name: test-asan
 on:
   pull_request:
     types: [opened, synchronize, reopened, ready_for_review]
-    paths-ignore:
-      - '.mailmap'
-      - '**.md'
-      - 'AUTHORS'
-      - 'doc/**'
+    paths:
```

Comment:
I suggest something like this instead:
```yml 
  paths-ignore:
    - .mailmap
    - **.md
    - AUTHORS
    - doc/**
    - .github/**
    - '!.github/workflows/test-asan.yml'
```
It's more readeable IMO.

Reply:
See https://github.com/nodejs/node/pull/40928#discussion_r754737190.

Reply:
I had already tested `!` in `paths-ignore` for another PR and it works.

Reply:
So the way I had it the first time would have worked fine. Harumph.